### ISSUE TITLE
Twint can't get retweets

### ISSUE THREAD

**Issue by alexandreteles** [twintproject/twint#484]
### Initial Check
> If the issue is a request please specify that it is a request in the title (Example: [REQUEST] more features). If this is a question regarding 'twint' please specify that it's a question in the title (Example: [QUESTION] What is x?). Please **only** submit issues related to 'twint'. Thanks.

- [x] Python version is 3.7.3;
- [x] Updated Twint with `pip3 install --user --upgrade -e git+https://github.com/twintproject/twint.git@origin/master#egg=twint`;
- [x] I have searched the issues and there are no duplicates of this issue/question/request.

### Command Ran
```python
import twint

searchParameters = twint.Config()

searchParameters.User = "madrilenha"
searchParameters.Retweets = True

twint.run.Search(searchParameters)
```

OR

```python
import twint

searchParameters = twint.Config()

searchParameters.User = "madrilenha"

twint.run.Search(searchParameters)
```

### Description of Issue
When trying to dump an user profile that contains only retweets twint will always fail with:

```
CRITICAL:root:twint.run:Twint:Feed:Tweets_known_error:'items_html'
'items_html' [x] run.Feed
[!] if get this error but you know for sure that more tweets exist, please open an issue and we will investigate it!
```

### Environment Details
Using twint on Windows, installed using git on a conda install.

**Comment by DenseLance** [comment]
Hello there,
It seems like there is no module named "user". What you are looking for is "Username" instead.
The follow code shows the scraping of retweets from the user's timeline named "madrilenha".

So for example,
```
import twint

searchParameters = twint.Config()
searchParameters.Username = "madrilenha"
searchParameters.Retweets = True
searchParameters.Format = "Post ID: {id} | Username: {username} | Date: {date}"

twint.run.Search(searchParameters)
```
I did formatting such that the posts are easier to read. To find the retweets of user, set `searchParameters.Retweets` as True.  I am not sure if this is correct, so do correct me if I am wrong.

Output:
```
Post ID: 1154951441297137664 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154946397881475073 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154944549778788352 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154942346301267968 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154940699244138496 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154936203608764416 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154913003315302401 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154900470458146818 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154898982142906373 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154893824956731392 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154876658052149248 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154871376911749121 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154807132157632513 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154805185371418624 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154803308529442817 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154801311076028417 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154799237680513024 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154798193630240768 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154796092929859584 | Username: madrilenha | Date: 2019-07-27
Post ID: 1154794316646309891 | Username: madrilenha | Date: 2019-07-27
```
There is still more below that I didn't paste here.

**Comment by pielco11** [comment]
You should run `.Profile` instead, since retweets are scraped in the user's timeline. You can't search for retweets sent by an user, so you don't have to use `.Search`

```python
import twint

searchParameters = twint.Config()
searchParameters.Username = "madrilenha"
searchParameters.Retweets = True
searchParameters.Format = "Post ID: {id} | Username: {username} | Date: {date}"

twint.run.Profile(searchParameters)
```

**Comment by alexandreteles** [comment]
> Hello there,
> It seems like there is no module named "user". What you are looking for is "Username" instead.
> The follow code shows the scraping of retweets from the user's timeline named "madrilenha".

I'm so sorry, I was pissed after a 12h sprint so I'm very pleased to be kind comment. Thank you. I got what I was doing wrong the next day after I've opened this issue and could only get back to it today. Again, thank you very much.

Is there any way to get all retweets to a given tweet using twint or is it limited on Twitter web the same way it is in the API?
